### DISCUSSION TITLE
How to clone an open SSH session

### DISCUSSION THREAD

**Discussion by lcmgithub** [kingToolbox/WindTerm#559] [Q&A] [answered]
Open a session, then clone a new session on the open session.

**Comment by kingToolbox** [comment, marked as answer]
Sorry for the late reply.

There is a menu item `Duplicate View` in the right-click menu of the session tab, and you can duplicate the session after clicking it.

**Reply by lcmgithub** [reply]
thank you very much!
Sorry for taking up your time.

**Reply by kingToolbox** [reply]
You are welcome. :smile:
